packages/smooth_app/lib/helpers/app_helper.dart

```diff
@@ -6,6 +6,8 @@ class AppHelper {
   static String getAssetPath(String asset) {
     if (asset.startsWith('/')) {
       asset = asset.substring(0);
```

Comment:
Just curious: you mean `asset = asset.substring(1)`, right?

Reply:
You're totally right, but it was somehow working in the app ^^